src/virt-install

```diff
@@ -286,17 +286,17 @@ try:
         # https://bugzilla.redhat.com/show_bug.cgi?id=1659242
 
         # set correct location of the kernel image and initrd
-        arch = get_basearch()
-        if arch == "x86_64" or arch == "i386":
+        basearch = get_basearch()
+        if basearch == "x86_64" or arch == "i386":
```

Comment:
@dustymabe Omission? ```arch == "i386"``` shouldn't it be ```basearch == "i386"``` too?

Reply:
Really, we should just drop that... Fedora doesn't even build for 32-bit x86 anymore right?

Reply:
https://github.com/coreos/coreos-assembler/pull/804

Reply:
good catch @jcajka - thanks for pointing that out